e2e/helpers.ts

```diff
@@ -708,6 +748,102 @@ export async function importHardwareWalletFlow(
   await findElementByText(driver, 'Rainbow is ready to use');
 }
 
+export async function importWalletFlowUsingKeyboardNavigation(
+  driver: WebDriver,
+  rootURL: string,
+  walletSecret: string,
+  secondaryWallet = false as boolean,
+) {
+  if (secondaryWallet) {
+    await goToPopup(driver, rootURL);
+    await executePerformShortcut({ driver, key: 'w' });
+    await findElementByTestIdAndClick({ id: 'add-wallet-button', driver });
+    await executePerformShortcut({
+      driver,
+      key: 'ARROW_DOWN',
+      timesToPress: 3,
+    });
+    await executePerformShortcut({ driver, key: 'ENTER' });
+  } else {
+    await goToPopup(driver, rootURL);
+    await executePerformShortcut({
+      driver,
+      key: 'ARROW_DOWN',
+      timesToPress: 2,
+    });
+    await executePerformShortcut({ driver, key: 'ENTER' });
+    await executePerformShortcut({
+      driver,
+      key: 'ARROW_DOWN',
+      timesToPress: 2,
+    });
+    await executePerformShortcut({ driver, key: 'ENTER' });
+  }
+
+  // ok
+  const isPrivateKey =
+    walletSecret.substring(0, 2) === '0x' && walletSecret.length === 66;
```

Comment:
this is fine, just fyi string also has a `startsWith` method you can use like `walletSecret.startsWith('0x')`